src/data/courseDependenciesData.js

```diff
@@ -36,6 +38,9 @@ const courseDependencyData = (courseSlug) =>
         ],
         pairWithResources: [
           {
+            id: '414202',
+            type: 'playlist',
+            guid: '553c',
```

Comment:
not sure if guid is right here.. I grabbed the end of the playlist slug. `id` probably works?